test/mix/tasks/compile.make_test.exs

```diff
@@ -6,6 +6,11 @@ defmodule Mix.Tasks.Compile.ElixirMakeTest do
 
   @fixture_project Path.expand("../../fixtures/my_app", __DIR__)
 
+  @exec_notfound_msg """
```

Comment:
We don't need to check for the full message, just for something like `~r/not found in the path/`.